src/schema/v2/artist/helpers.ts

```diff
@@ -0,0 +1,112 @@
+import { compact } from "lodash"
```

Comment:
I think it makes a ton of sense to move a bunch of this logic into a helper file and out of the insights file, nice work!